### PR TITLE
Update layout to bootstrap 4

### PR DESCRIPTION
**Pull request by Katrienvh**
Update markup to bootstrap 4
Use a macro because the markup of a quote is always the same

### CODE REVIEW

**Review by StijnVrolijk — APPROVED** [review]

File: src/Frontend/Modules/Quotes/Layout/Widgets/AllQuotes.html.twig [hunk 1]
```diff
@@ -1,4 +1,7 @@
-{% for quote in quotes %}
-  <blockquote>{{ quote.quote|raw }}</blockquote>
-  <p>{{ quote.name }}</p>
-{% endfor %}
+{% import 'Modules/Quotes/Layout/Templates/Quote.html.twig' as macro %}
```
Comment (mathiashelin) [inline src/Frontend/Modules/Quotes/Layout/Widgets/AllQuotes.html.twig]: `macro` is too general as a name for your macro. What if you also need to import a social media buttons macro?

File: src/Frontend/Modules/Quotes/Layout/Templates/Quote.html.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% macro quote(quote) %}
```
Comment (mathiashelin) [inline src/Frontend/Modules/Quotes/Layout/Templates/Quote.html.twig:1]: Maybe use `parseQuote()`

File: src/Frontend/Modules/Quotes/Layout/Widgets/RandomQuote.html.twig [hunk 3]
```diff
@@ -1,23 +1,7 @@
...
-    {% if not quote.image %}
-      <div class="col-sm-12">
-        {{ quote.quote|raw }}
-        -- {{ quote.name }}
-      </div>
-    {% endif %}
-    </div>
-  </blockquote>
-{% endif %}
+{% import 'Modules/Quotes/Layout/Templates/Quote.html.twig' as macro %}
```
Comment (mathiashelin) [inline src/Frontend/Modules/Quotes/Layout/Widgets/RandomQuote.html.twig]: Same, `macro` is to general

File: src/Frontend/Modules/Quotes/Layout/Widgets/SingleQuote.html.twig [hunk 4]
```diff
@@ -1,22 +1,7 @@
...
-    {% if not quote.image %}
-      <div class="col-sm-12">
-        {{ quote.quote|raw }}
-        -- {{ quote.name }}
-      </div>
-    {% endif %}
-    </div>
-  </blockquote>
-{% endif %}
+{% import 'Modules/Quotes/Layout/Templates/Quote.html.twig' as macro %}
```
Comment (mathiashelin) [inline src/Frontend/Modules/Quotes/Layout/Widgets/SingleQuote.html.twig]: `macro` is too general.

**Review by mathiashelin — CHANGES_REQUESTED** [review]

File: src/Frontend/Modules/Quotes/Layout/Templates/Quote.html.twig (hunk 2 above)
Comment (Katrienvh) [reply src/Frontend/Modules/Quotes/Layout/Templates/Quote.html.twig]: fixed in b0c933b

File: src/Frontend/Modules/Quotes/Layout/Widgets/AllQuotes.html.twig (hunk 1 above)
Comment (Katrienvh) [reply src/Frontend/Modules/Quotes/Layout/Widgets/AllQuotes.html.twig]: fixed in b0c933b

File: src/Frontend/Modules/Quotes/Layout/Widgets/RandomQuote.html.twig (hunk 3 above)
Comment (Katrienvh) [reply src/Frontend/Modules/Quotes/Layout/Widgets/RandomQuote.html.twig]: fixed in b0c933b

File: src/Frontend/Modules/Quotes/Layout/Widgets/SingleQuote.html.twig (hunk 4 above)
Comment (Katrienvh) [reply src/Frontend/Modules/Quotes/Layout/Widgets/SingleQuote.html.twig]: fixed in b0c933b

**Review by mathiashelin — APPROVED** [review]

**Pull request merged**